nextest-runner/default-config.toml

```diff
@@ -97,6 +97,9 @@ slow-timeout = { period = "60s" }
 # See <https://nexte.st/docs/features/leaky-tests> for more information.
 leak-timeout = "100ms"
 
+# Stop all tests after the configured global timeout.
+global-timeout = "30y"
```

Comment:
Could this explain why it's 30 years? The explanation in `far_future` in `pausable_sleep.rs` can be copied or adapted here.